packages/kbn-journeys/journey/journey_ftr_harness.ts

```diff
@@ -240,9 +240,8 @@ export class JourneyFtrHarness {
       return await block();
     }
 
-    const span = this.apm?.startSpan(name, type ?? null, {
-      childOf: this.currentTransaction,
-    });
+    const span = this.currentTransaction.startSpan(name, type ?? null);
+
```

Comment:
Thank you for the fix. I need to run a few pipelines to make sure we are able to extract APM transactions for scalability test simulation